extensions/ql-vscode/src/vscode-tests/no-workspace/query-test.ts

```diff
@@ -168,6 +168,7 @@ describe('using the query server', function() {
             localChecking: false,
             noComputeGetUrl: false,
             noComputeToString: false,
+            computeDefaultStrings: true
```

Comment:
Can you add a new `QueryTestCase` to exercise this?
The simplest I can think of is a query that looks like this:

```ql
newtype TUnit = MkUnit()

class Unit extends TUnit {
  Unit() { this = MkUnit() }

  string toString() { none() }
}

from Unit u
select u
```

Existing behaviour: this produces 0 results.
New behaviour: this produces 1 result, whose text is `(no string representation)`.

Reply:
I added that integration test to the list. #676

Currently, even the basic query test is failing. Need to work on that.

Reply:
Sure. For this particular test, we already have tests in `query-test` to run a simple query and get the raw results out, so I suggest adding a case in this PR if possible without needing the new integration test machinery.

Reply:
Ah...I see. OK. I will add that tomorrow.

Reply:
So, those tests have never been running in CI. Our scripts match on `*.test.ts`, but that file is `query-test.ts`. I will fix.

Reply:
😱